### PR TITLE
feat: Update create-user script to create operator users correctly

### PR DESCRIPTION
**Pull request by alainajaffer**

### CODE REVIEW

File: cli-helpers/README.md [hunk 1]
```diff
@@ -53,7 +64,24 @@ DESCRIPTION
...
 ```
 
-_See code: [dist/cli-helpers/src/commands/create-org/index.ts](https://github.com/Department-for-Transport-Disruptions/create-disruptions-data/blob/v0.0.0/dist/cli-helpers/src/commands/create-org/index.ts)_
+## `cdd-helpers create-operator-org`
+
+Create operator organisation
+
+```
+USAGE
+  $ ./bin/run.js create-operator-org --name <value> --adminAreaCodes <value> --stage <value>
```
Comment (DeannaSharma) [inline cli-helpers/README.md:73]:
```suggestion
  $ ./bin/run.js create-operator-org --name <value> --nocCodes <value> --stage <value> --orgId <value>
```

File: cli-helpers/README.md [hunk 2]
```diff
@@ -53,7 +64,24 @@ DESCRIPTION
...
+
+FLAGS
+  --nocCodes=<value>        (required) Comma-separated list of noc codes
+  --name=<value>            (required) Name of operator organisation
```
Comment (DeannaSharma) [inline cli-helpers/README.md:77]:
```suggestion
  --name=<value>            (required) Name of operator organisation
  --orgId-<value>.          (required) ID for organisation that the operator organisation belongs to
```

File: cli-helpers/README.md [hunk 3]
```diff
@@ -73,6 +101,7 @@ FLAGS
   --lastName=<value>   Last name of user
   --orgId=<value>      ID for organisation that user belongs to
   --poolId=<value>     ID of user pool to add user to
+  --operatorOrgId=<value> Operator organisation ID
```
Comment (DeannaSharma) [inline cli-helpers/README.md:104]: Might be good to add `--operatorOrgId=<value>` to the USAGE command on line 66 too

Comment (alainajaffer) [reply cli-helpers/README.md:105]: Not sure it is required on that line

File: cli-helpers/README.md (hunk 2 above)
Comment (alainajaffer) [reply cli-helpers/README.md:77]: done

File: cli-helpers/README.md (hunk 1 above)
Comment (alainajaffer) [reply cli-helpers/README.md]: done

**Review by DeannaSharma — APPROVED** [review]

**Pull request merged**